src/_scss/components/_equityMainCard.scss

```diff
@@ -0,0 +1,124 @@
+.equity-main-card-content {
+  border-radius: 8px;
+  margin: $global-margin * 2 0;
+  display: block;
+
+  @media (max-width: $small-screen) {
+    margin: 0;
+  }
+
+  @media (min-width: $medium-screen) {
+    align-items: center;
+    width: auto;
+    min-height: 500px;
```

Comment:
is this min-height doing anything here?

Reply:
nope, had that before i had the height percentage there so it is no longer doing anything